Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean

```diff
@@ -38,13 +38,13 @@ namespace AlgebraicGeometry
 topological map is a closed embedding and the induced stalk maps are surjective. -/
 @[mk_iff]
 class IsClosedImmersion {X Y : Scheme} (f : X ⟶ Y) : Prop where
-  base_closed : ClosedEmbedding f.1.base
+  base_closed : ClosedEmbedding f.base
   surj_on_stalks : ∀ x, Function.Surjective (f.stalkMap x)
 
 namespace IsClosedImmersion
 
```

Comment:
I can't comment on unchanged code, but in `iff_isPreimmersion` and in `of_isPreimmersion` below there are two more `f.1.base` which should be `f.base`